src/WaveClip.cpp

```diff
@@ -1789,3 +1712,233 @@ const wxString& WaveClip::GetName() const
 {
    return mName;
 }
+
+void WaveClip::ClearSequence(double t0, double t1)
```

Comment:
Move this function, perhaps, so it is just after WaveClip::Clear, so the diff of the two versions is a bit smaller, more corresponding unchanged lines.